instrumentation-api/src/main/java/io/opentelemetry/instrumentation/api/MoreAttributes.java

```diff
@@ -21,4 +21,11 @@
   public static final String HTTP_FRAGMENT = "http.fragment.string";
 
   public static final String USER_NAME = "user.principal";
+  public static final String QUERY_PAGE_SIZE = "query.pageSize";
```

Comment:
I think in parallel with the spec change we could proceed if this is scoped to cassandra. How about moving these into the cassandra package and prefixing with that for now? @trask what do you think?

Reply:
> I think in parallel with the spec change we could proceed if this is scoped to cassandra

Once a spec PR is opened, I'm good merging something here that implements the initial spec PR (and we can sync it up later with whatever ends up getting approved/merged in the spec).

> How about moving these into the cassandra package and prefixing with that for now?

👍